acceptance_tests/features/view_survey_list.feature

```diff
@@ -0,0 +1,19 @@
+Feature: View survey list
+  As a Respondent
+  I need to see collection exercise(s) only if the the event 'Go-Live' has occurred/passed
+  So that collection exercises are not visible to respondents before they should be
+
+  Background: User already logged in
+    Given the respondent is signed into their account
+
+  @us060-filterCEByGoLive_s01
+  Scenario: Respondent can view collection exercise after the Go Live Event
+    Given the go live event for the collection exercise has passed
+    When access to do list in my surveys
```

Comment:
Using first person here and then third person in the 'then'.
When the respondent accesses the surveys to do list?

Reply:
I agree with this, this happened because the when step is taken from the `add_a_survey` steps file. 
What I think really need here is to go through all the features files and standardise the language we're using. It's inconsistent and that means we have to redefine steps often.
I'll change these here though to at least be consistent in this file :+1: